### DISCUSSION TITLE
[request] CONHOST.EXE 10.0.19041.4355 source

### DISCUSSION THREAD

**Discussion by 0wwafa** [microsoft/terminal#18065] [Q&A] [answered]
On my windows version, I have CONHOST.EXE version **10.0.19041.4355**

Where can I get the source code of this specific version?

**Comment by lhecker** [comment, marked as answer]
This commit from our repository is reasonably close to what shipped in that version of Windows: https://github.com/microsoft/terminal/tree/d4d59fa3395012ca37ba12665da4ec11c7dcf9cb

It is not possible to give you the exact source code, because conhost is not open source. Only OpenConsole is.
